ethstaker_deposit/utils/validation.py

```diff
@@ -427,9 +427,46 @@ def validate_bls_to_execution_change_keystore(validator_index: str,
     )
 
     domain = compute_bls_to_execution_change_keystore_domain(
-        fork_version=chain_settings.GENESIS_FORK_VERSION,
-        genesis_validators_root=chain_settings.GENESIS_VALIDATORS_ROOT
+        fork_version=chain_setting.GENESIS_FORK_VERSION,
+        genesis_validators_root=chain_setting.GENESIS_VALIDATORS_ROOT
     )
 
     signing_root = compute_signing_root(message, domain)
     return bls.Verify(bls_pubkey, signing_root, bls_signature)
+
+
+#
+# Devnet Chain Setting Validation
+#
+
+def validate_devnet_chain_setting(ctx: click.Context, param: Any, value: Optional[str]) -> Optional[str]:
+    if value is None:
+        return None
+
+    trimmed_value = value[:400]
```

Comment:
What is the purpose of this?

Reply:
It's because of this warning from https://docs.python.org/3/library/json.html:
> Warning Be cautious when parsing JSON data from untrusted sources. A malicious JSON string may cause the decoder to consume considerable CPU and memory resources. Limiting the size of data to be parsed is recommended.